pyamg/aggregation/smooth.py

```diff
@@ -904,7 +904,7 @@ def gmres_prolongation_smoothing(A, T, B, BtBinv, Sparsity_Pattern, maxiter,
 def energy_prolongation_smoother(A, T, Atilde, B, Bf, Cpt_params,
                                  krylov='cg', maxiter=4, tol=1e-8,
                                  degree=1, weighting='local',
-                                 Pfilter=None):
+                                 Pfilter=None, prefilter=None):
```

Comment:
I think we should remove "row-wise." In that case does it make sense for prefilter and postfilter be dictionaries with possible arguments 'k' and 'theta'?